includes/blocks/class-sensei-blocks.php

```diff
@@ -63,4 +63,43 @@ public function sensei_block_categories( $categories, $post ) {
 			]
 		);
 	}
+
+	/**
+	 * Register Sensei block. It's a wrapper for `register_block_type` or
+	 * `register_block_type_from_metadata`, allowing to filter the args.
+	 *
+	 * @param string $block_name     Block name.
+	 * @param array  $block_args     Block arguments.
+	 * @param string $file_or_folder Path to the JSON file with metadata definition for
+	 *                               the block or path to the folder where the `block.json`
+	 *                               file is located. The block will be registered using
+	 *                               `register_block_type_from_metadata` if it's defined.
+	 */
+	public static function register_sensei_block( $block_name, $block_args, $file_or_folder = null ) {
+		/**
+		 * Filter the args of the Sensei blocks.
+		 *
+		 * In WordPress versions 5.5 and later, block type arguments can be filtered by using register_block_type_args
+		 * instead. This filter exists to support earlier versions only.
+		 *
+		 * Notice that for blocks being registered using the `$file_or_folder`, this filter runs before the
+		 * `register_block_type_from_metadata`.
+		 *
+		 * @since 3.6.0
+		 * @hook sensei_block_type_args
+		 * @see register_block_type
+		 * @see register_block_type_from_metadata
+		 * @see includes/blocks/compat.php
+		 *
+		 * @param {array}  $args The block arguments as defined by register_block_type.
+		 * @param {string} $args Block name.
```

Comment:
Param variable name needs to be updated.

Reply:
Oops! Fixed here: https://github.com/Automattic/sensei/pull/3782/commits/cb156c1ec347b7f2df9b3c2355e19673d2149798